### ISSUE TITLE
spiegel.de: anti-adblock

### ISSUE THREAD

**Issue by fv043** [uBlockOrigin/uAssets#7855]
### URL(s) where the issue occurs
`https://www.spiegel.de/`

### Describe the issue
Spiegel.de redirects to an anti-adblock page that demands the user to formally agree to ads and whitelist the site, or else subscribe.  If you whitelist the site, then click "Accept", it sets cookies that enable accessing the content.  

I tried the filters suggested in #7298 but they didn't do anything.

 

### Screenshot(s)
![spiegel de](https://user-images.githubusercontent.com/36488914/91742082-a8980680-eb83-11ea-82aa-c5ce723393b9.png)

### Versions

- Browser/version: Pale Moon 28.12.0
- uBlock Origin version: uBlock-for-firefox-legacy 1.16.4.25

### Settings

1. Default uBO settings with new Pale Moon profile;
2. My regular profile with a number of additonal lists and advance mode enabled.

### Notes
Blocking 1st party scripts allowed a simplified, no images, version of the page to load.  
Actually, blocking either of these two scripts did that:  
`https://cdn.prod.www.spiegel.de/public/shared/generated/js/header.8122618b598d496bb8b4.js`  
or
`https://cdn.prod.www.spiegel.de/public/shared/generated/js/commons.d84e4d37a0dda95357b1.js`

**Comment by uBlock-user** [comment]
That's server-sided cookie consent page, not anti-adblock. Force update EasyPrivacy and then click on Accept.

**Comment by fv043** [comment]
> Force update EasyPrivacy and then click on Accept.

It did become possible to click _Accept_ after updating EasyPrivacy.  
Before updating it, clicking _Accept_ did nothing unless the site was first whitelisted.  So, thanks.

**Comment by jansohn** [comment]
> 
> 
> That's server-sided cookie consent page, not anti-adblock. Force update EasyPrivacy and then click on Accept.

I've tried this after resetting uBlock Origin to default settings. Unfortunately it's not working and I still get the following message when I press Accept:

> We were unfortunately unable to place the cookie that is necessary for the use of our site. Is your computer connected to the internet through a company network? We recommend that you ask your IT department for an exception for DER SPIEGEL content. Have you configured a browser extension or the browser itself such that cookies are blocked? Please add DER SPIEGEL to your list of exceptions.
Should you continue to have problems, you can find more information here. 

Anything else I can try? Should I open a new issue? I'm on Firefox 80.0.1 (64-bit) and uBlock Origin 1.29.2.

**Comment by okiehsch** [comment]
Works fine on my end.
Can you open the logger and check if `privacy-mgmt.com` requests are still blocked on your end?
Open the [logger[(https://github.com/gorhill/uBlock/wiki/The-logger) go to `spiegel.de` and filter the putput by typing 
"privacy" in the box.
![image](https://user-images.githubusercontent.com/16838044/92664378-1b337f80-f304-11ea-9e1b-eef20a9d6637.png)

**Comment by jansohn** [comment]
@okiehsch they don't seem to be blocked on my end...

![image](https://user-images.githubusercontent.com/7804514/92791611-1971c680-f3ad-11ea-8221-d970a2a87e6e.png)

**Comment by okiehsch** [comment]
Hm, so uBO is not responsible for the "unable to place the cookie" message on your end.

Do you have any other extensions running or do you have cookies blocked via Firefox directly?

**Comment by jansohn** [comment]
I disabled/uninstalled a couple of other extensions (NoScript, Privacy Badger, HTTPS Everywhere) which might interfere but that didn't change anything. I haven't changed anything in the Firefox settings.

Console output shows the blocked XHR and CORS problems but I don't know why they (still) occur:
![image](https://user-images.githubusercontent.com/7804514/92974316-36c69380-f486-11ea-99be-87e4d22e67d5.png)

**Comment by okiehsch** [comment]
Can you use the site if you disable all extensions?
Looks like a Firefox issue have you tested with a different browser?

**Comment by jansohn** [comment]
Well, it works if I disable uBlock Origin...

**Comment by uBlock-user** [comment]
Instead of disabling uBO, disable all other extensions and then test with default settings.

**Comment by jansohn** [comment]
> 
> 
> Instead of disabling uBO, disable all other extensions and then test with default settings.

I've re-installed Firefox and only added uBlock Origin as an extension. Updated all filter lists and retried. Same problem...

I can see the blocked XHR in the logger:
![image](https://user-images.githubusercontent.com/7804514/93003571-2672ef00-f540-11ea-92d9-0138b35bb4d2.png)

**Comment by uBlock-user** [comment]
Updated the CNAME exception filter, force update unbreak list in few mins.

**Comment by jansohn** [comment]
Working now, thanks!
